src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerConfig.java

```diff
@@ -80,12 +80,14 @@
       + "the offset to the latest offset</li><li>none: throw exception to the consumer if no previous offset is found "
       + "for the consumer's group</li><li>anything else: throw exception to the consumer.</li></ul>";
 
-  private static final String SKIP_RECORD_ON_EXCEPTION_DOC = "Skip the record with RecordProcessingException. "
-      + "This is to allow the users who do are willing to ignore the problematic record to continue consuming without "
+  private static final String SKIP_RECORD_ON_SKIPPABLE_EXCEPTION_DOC = "Skip the record with SkippableException. "
```

Comment:
Might want to say something like:
"When processing a record and a SkippableException is thrown then a log message is generated and the exception is ignored.  The code will processed to process records as if the exception never happened.  The record which caused the SkippableException will be lost."